microprofile.jdt/org.eclipse.lsp4mp.jdt.tp/org.eclipse.lsp4mp.jdt.target

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<?pde version="3.8"?>
+<target name="Running Platform">
+	<locations>
+		<location type="Target" uri="https://raw.githubusercontent.com/eclipse/eclipse.jdt.ls/master/org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target" />
```

Comment:
Is there a better way to reference this target file? Would it be better to reference the target file from a tagged release?

Reply:
The only disadvantage to this is rebuilding a tag would never work as expected as it would always use the latest :\ .

So either we always refer to a tagged release, or in the case where there are breaking changes, some commit reference.